prac_06/guitars.py

```diff
@@ -0,0 +1,32 @@
+"""
+CP1404 Prac06 - Guitars
+Sam Butters
+Time started: 15:40
+Time finished: 16.21
+"""
+from prac_06.guitar import Guitar
+
+guitars = []
+
+name = input("Name: ")
+while name != "":
+    year = int(input("Year: "))
+    cost = float(input("Cost: "))
+    guitar = Guitar(name, year, cost)
+    guitars.append(guitar)
+    print(f"{guitar} added.")
+    name = input("Name: ")
+
+guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+max_name_length = max([len(guitar.name) for guitar in guitars])
+max_cost_length = max(len(f"${guitar.cost:.2f}") for guitar in guitars)
```

Comment:
good, if you add the comma after the colon you will get the cost format from the example (16,035.40
e.g. {guitar.cost:,.2f}